### DISCUSSION TITLE
Support Apache IoTDB in data source

### DISCUSSION THREAD

**Discussion by mychaow** [grafana/grafana#38341] [Data Sources]
**What would you like to be added**:
Support Apache IoTDB as data source

**Why is this needed**:
Apache IoTDB is an IoT native database with high performance for data management and analysis, deployable on the edge and the cloud. By supporting iotdb as the data source, we can make grafana more functional as a data visualization solution for the Internet of things.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
